src/Resources/Resources.Test/ScenarioTests/ResourceTestRunner.cs

```diff
@@ -54,21 +54,23 @@ protected ResourceTestRunner(ITestOutputHelper output)
                 {
                     {"Microsoft.Azure.Management.ResourceManager.ResourceManagementClient", "2016-07-01"},
                 })
+                .WithMockContextAction(() =>
+                {
+                    var credentials = HttpMockServer.Mode != HttpRecorderMode.Playback
+                        ? new Func<SubscriptionCloudCredentialsAdapter>(() =>
+                            {
+                                var testEnvironment = TestEnvironmentFactory.GetTestEnvironment();
+                                return new SubscriptionCloudCredentialsAdapter(
```

Comment:
If possible, we should remove their dependency on this - please look to see if we can quickly have them depend on an IToken or a ServiceClientCredentials object - please take a look at how long it would take to do this

Reply:
This should take a day to fix and test -- opened an issue here: https://github.com/Azure/azure-powershell/issues/8678